packages/did-provider-key/src/SphereonKeyDidProvider.ts

```diff
@@ -67,7 +67,7 @@ export class SphereonKeyDidProvider extends AbstractIdentifierProvider {
 
     const key = await importProvidedOrGeneratedKey(
       {
-        kms: kms ?? this.kms,
+        kms: kms ?? this.kms ?? '',
```

Comment:
Rather have a ts-ignore here. Our KMS handles it correctly when no KMS is provided. Using an empty string will result in errors